tools/boards.txt.py

```diff
@@ -1682,6 +1682,17 @@ def sdk ():
 
 ################################################################
 
+def float_in_iram ():
+    return { 'iramfloat': collections.OrderedDict([
+                        ('.menu.iramfloat.no', 'Not allowed in ISR (more IRAM)'),
+                        ('.menu.iramfloat.no.build.iramfloat', '-DFP_IN_IROM'),
+                        ('.menu.iramfloat.yes', 'Allowed in ISR (less IRAM)'),
+                        ('.menu.iramfloat.yes.build.iramfloat', '-DFP_IN_IRAM'),
```

Comment:
May lose IRAM too in the 1st? Just mention in the 2nd option

: Not allowed in ISR
: Allowed in ISR (+1KiB IRAM) *(or however much, we definitely know here)*

Reply:
Or, as VTABLES one. Just to signify they don't go away, just change mapping

: Not allowed in ISR (Flash)
: Allowed in ISR (IRAM)